_locales/en/messages.json

```diff
@@ -137,6 +137,13 @@
   "typingIndicatorsSettingDescription": "See and share when messages are being typed (applies to all sessions).",
   "typingIndicatorsSettingTitle": "Typing Indicators",
   "zoomFactorSettingTitle": "Zoom Factor",
+  "pruneSettingTitle": "Prune Old Open Group Messages",
+  "pruneSettingDescription": "Prune messages older than 6 months from opengroups on start",
```

Comment:
I suggest replacing all `opengroups` text that is seen in the UI with `Open Groups` to be consistent.